bootstrap4/.js/src/test/scala/io/udash/bootstrap/datepicker/UdashDatePickerTest.scala

```diff
@@ -338,3 +375,9 @@ private trait JQueryDatePickerExt extends JQuery {
   def datetimepicker(function: String): js.Any = js.native
   def datetimepicker(option: String, value: js.Any): JQueryDatePickerExt = js.native
 }
+
+@js.native
+private trait MomentDate extends js.Any {
+  /** @return picker input string (currently set date formatted according to provided `format` string)  */
```

Comment:
Is there a doc link?

Reply:
Internal [`_i` property](https://momentjs.com/guides/#/lib-concepts/internal-properties/) replaced with a call to [`creationData`](https://momentjs.com/docs/#/parsing/creation-data/).